app/components/earthworks/other_options_component.rb

```diff
@@ -0,0 +1,11 @@
+module Earthworks
+  # Display other options section in sidebar
+  class OtherOptionsComponent < ViewComponent::Base
+    attr_reader :document
+
+    def initialize(document:)
+      @document = document
```

Comment:
I think this is redundant, given the `attr_reader` ?

Reply:
I removed this component entirely, based on discussions with @dnoneill .